components/_pages/SignIn/constants/index.ts

```diff
@@ -9,6 +9,10 @@ export const INPUTS: Input<'username' | 'password'>[] = [
         value: true,
         message: 'This field is required',
       },
+      pattern: {
+        value: /^[a-zA-Z0-9]+$/,
+        message: '',
```

Comment:
if you create rules, need add validation error message.
I this on the login page we don't need validate this field